ov

```diff
@@ -0,0 +1,141 @@
+#! /usr/bin/env bash
+
+set -e
+
+# https://stackoverflow.com/a/1482133
+OPENVERSE_PROJECT="$(dirname "$(readlink -f -- "$0")")"
+export OPENVERSE_PROJECT
+
+dev_env="$OPENVERSE_PROJECT"/docker/dev_env
+
+is_yes() {
+  [[ ${1^^} == "Y" || ${1^^} == "YES" ]] && echo "true" || echo "false"
+}
```

Comment:
bash -c "bash <(curl -fsSL https://raw.githubusercontent.com/WordPress/openverse/e9378ee25f5643124aad6cc785abe2126b6d1148/ov)" prints everything out and stops with this:
```
Would you like to bootstrap the Openverse development environment? (nY): y
/dev/fd/63: line 12: ${1^^}: bad substitution
```